app/controllers/create.js

```diff
@@ -25,16 +26,20 @@ export default Controller.extend(EventWizardMixin, {
           }
           RSVP.Promise.all(promises)
             .then(() => {
-              this.transitionToRoute('events.view.edit.sponsors', data.id);
+              this.set('isLoading', false);
```

Comment:
The loader is toggled in then & catch clause because the finally gets executed before all the requests are resolved. These forms have multiple requests associated, which must be resolved before the loader turns off.